### PR TITLE
QAE-869 - E2E test case for off canvas menu margin

### PR DESCRIPTION
**Pull request by SimranChougule1**
### Description

Header builder → Off canvas → off canvas menu → Design → Margin option

### Screenshots

https://share.getcloudapp.com/GGul2ZgG

### How has this been tested?

Run test: npm run test:e2e:interactive -- tests/e2e/specs/customizer/header-builder/header-types/mobile-header/off-canvas-menu/margin.test.js

### Checklist:
- [x] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: tests/e2e/specs/customizer/header-builder/header-types/mobile-header/off-canvas-menu/margin.test.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+		}
+		await page.goto( createURL( '/' ), {
+			waitUntil: 'networkidle0',
+		} );
+		await setBrowserViewport( 'medium' );
+		await page.click( '.main-header-menu-toggle' );
+		await expect( {
+			selector: '.ast-builder-menu-mobile .main-header-menu',
+			property: 'margin-top',
+		} ).cssValueToBe( `${ offCanvasMargin[ 'header-mobile-menu-margin' ].tablet.top }${ offCanvasMargin[ 'header-mobile-menu-margin' ][ 'tablet-unit' ] }` );
```
Comment (bsfsnehal) [inline tests/e2e/specs/customizer/header-builder/header-types/mobile-header/off-canvas-menu/margin.test.js:41]: @SimranChougule1 test case is failing here,please execute at your local-end

Comment (SimranChougule1) [reply tests/e2e/specs/customizer/header-builder/header-types/mobile-header/off-canvas-menu/margin.test.js:41]: Changed selector please check once at your local end

Comment (bsfsnehal) [reply tests/e2e/specs/customizer/header-builder/header-types/mobile-header/off-canvas-menu/margin.test.js:42]: @SimranChougule1 this test case failing at local,please check.

Comment (SimranChougule1) [reply tests/e2e/specs/customizer/header-builder/header-types/mobile-header/off-canvas-menu/margin.test.js]: @bsfsnehal Resolved comments

**Review by bsfsnehal — APPROVED** [review]

**Review by GauravKhupse — APPROVED** [review]

**Pull request merged**